src/main/java/eu/europeana/metis/sandbox/domain/Record.java

```diff
@@ -75,10 +98,25 @@ public Language getLanguage() {
     return this.language;
   }
 
-  public String getContent() {
+  /**
+   * Content of the record
+   *
+   * @implNote Overwriting this field contents after construction could cause problems.
+   * <br />
+   * We are not making a copy of it because it
+   * is expensive and Record object is expected to be use as a non mutable object
+   */
+  public byte[] getContent() {
     return this.content;
   }
 
+  public String getContentString() {
```

Comment:
Lazy initialization, only if the content is required as a String it will be populated

Reply:
Added to [MET-2680](https://europeana.atlassian.net/browse/MET-2680) to be removed after the change